src/components/fields/MessageField.vue

```diff
@@ -64,11 +73,33 @@ export default {
 		},
 		title () {
 			return `Click to view ${this.getKeyName(`messageTypeDescriptor_${this.messageType}`)}`;
+		},
+		viewFormatTitle () {
+			return this.isViewAsUTF8 ? `View as default` : `View as utf-8`;
+		},
+		isHexString () {
+			return Convert.isHexString(this.value.payload);
```

Comment:
What happens if there is no payload?  I see the convert button when there is no message.